### ISSUE TITLE
accessSheet function sends incorrect arguments to `getAllAttrs`

### ISSUE THREAD

**Issue by ZachHaber** [Kurohyou/Roll20-Snippets#25]
https://github.com/Kurohyou/Roll20-Snippets/blob/c5eed3d6af3b844a6cef44379cc1c95f710580c3/K_Scaffold/Template%20v2/scaffold/accessSheet.js#L56

The call passes in `event` to `getAllAttrs` which doesn't take an `event` value.

```js
//accessSheet
const accessSheet = function(event){
  debug({funcs:Object.keys(funcs)});
  debug({event});
  getAllAttrs({event,callback:(attributes,sections,casc)=>{
    let trigger = attributes.getCascObj(event,casc);
    attributes.processChange({event,trigger,attributes,sections,casc});
  }});
};
```

```js
//getAllAttrs
const getAllAttrs = function({props=baseGet,sectionDetails=repeatingSectionDetails,callback}){
  getSections(sectionDetails,(repeats,sections)=>{
    getAttrs([...props,...repeats],(values)=>{
      const attributes = createAttrProxy(values);
      orderSections(attributes,sections);
      const casc = expandCascade(cascades,sections,attributes);
      callback(attributes,sections,casc);
    })
  });
};
```
```

**Comment by Kurohyou** [comment]
Thanks for the notice!

**Comment by Kurohyou** [comment]
Should be resolved by #30
